### DISCUSSION TITLE
Using wavesurfer.js in flask

### DISCUSSION THREAD

**Discussion by orbisDerekP** [katspaugh/wavesurfer.js#3294] [Q&A]
I'm using flask  on Python3.7 and want to add playback functionality. I'm using the dpkg URL via:
`<script src="https://unpkg.com/wavesurfer.js@7"></script>`

I have a 'div' for the content to load:
`<div id="waveform"> wavesurfer goes here </div>`

I think that the below code is where my breakdown is happening. My URL is valid as I've been able to load get playback e.g. HTMLAudio. I was hoping to get some advice one way or another, whether I'm missing something, or on a fool's errand. 
```{% block scripts %} 
<script>  
var wavesurfer = WaveSurfer.create({
container: document.getElementById('waveform');
waveColor: '#4F4A85';
progressColor: '#383351';
url: "{{ url_for('static', filename=wavdir) }}";
sampleRate: 8789;
});

wavesurfer.on('click', () => {
  wavesurfer.play()
});
</script>
{% endblock %}
```

**Comment by katspaugh** [comment]
For starters, your JS is syntactically not correct.

**Reply by orbisDerekP** [reply]
Good to know. What are you seeing?

**Reply by orbisDerekP** [reply]
pretty new to javascript

This still does not do anything except load the blank div. Is this formatting better, or worse...

```
{% block javascript %}
<script src="{{ url_for('static', filename='script/wavesurfer/dist/wavesurfer.js') }}">

const wavesurfer = WaveSurfer.create({
  container: document.body,
  waveColor: 'rgb(200, 0, 200)',
  progressColor: 'rgb(100, 0, 100)',
  url: '/examples/audio/audio.wav',
})

wavesurfer.on('click', () => {
  wavesurfer.play()
})


{% endblock %}
</html>>
```

**Reply by katspaugh** [reply]
This is invalid HTML.
I understand you’re new to JS, but this forum is for specific questions about wavesurfer.
You can try ChatGPT or Stackoverflow to get started.

**Reply by orbisDerekP** [reply]
good point, thanks.
